### PR TITLE
improve french translation

### PR DESCRIPTION
**Pull request by medyagh**
changing the out.Step's to be Dumb strings version so our `make extract` add them to our trasnlatiosn

### french before this PR (partial)
```
$ LC_ALL=fr mk start
😄  minikube v1.20.0 sur Darwin 11.2.3
✨  Choix automatique du pilote hyperkit. Autres choix: 
👍  Démarrage du noeud de plan de contrôle minikube dans le cluster minikube
🔥  Création de VM hyperkit (CPUs=2, Mémoire=4000MB, Disque=20000MB)...
🐳  Préparation de Kubernetes v1.20.2 sur Docker 20.10.6...
    ▪ Generating certificates and keys ...
    ▪ Booting up control plane ...
    ▪ Configuring RBAC rules ...
🔎  Verifying Kubernetes components...
    ▪ Using image gcr.io/k8s-minikube/storage-provisioner:v5
🌟  Enabled addons: storage-provisioner, default-storageclass
🏄  Done! kubectl is now configured to use "minikube" cluster and "default" namespace by default
```

### french after this PR (full  - includes Generating certificates and keys  ...)

```
$ LC_ALL=fr mk start
😄  minikube v1.20.0 sur Darwin 11.2.3
✨  Choix automatique du pilote hyperkit. Autres choix: 
👍  Démarrage du noeud de plan de contrôle minikube dans le cluster minikube
🔥  Création de VM hyperkit (CPUs=2, Mémoire=4000MB, Disque=20000MB)...
🐳  Préparation de Kubernetes v1.20.2 sur Docker 20.10.6...
    ▪ Générer des certificats et des clés ...
    ▪ Démarrage du plan de contrôle ...
    ▪ Configuration des règles RBAC ...
🔎  Vérification des composants Kubernetes...
    ▪ Utilisation de l'image gcr.io/k8s-minikube/storage-provisioner:v5
🌟  Addons activés: storage-provisioner, default-storageclass
🏄  Terminé ! kubectl est maintenant configuré pour utiliser "minikube" cluster et espace de noms "default" par défaut.
```

### CODE REVIEW

File: pkg/minikube/bootstrapper/kubeadm/kubeadm.go [hunk 1]
```diff
@@ -318,7 +317,17 @@ func outputKubeadmInitSteps(logs io.Reader, wg *sync.WaitGroup) {
...
-		out.Step(style.SubStep, nextStep.stepMessage)
+		// because the translation extract (make extract) needs simple strings to be included in translations we have to pass simple strings
+		if nextStepIndex == 0 {
+			out.Step(style.SubStep, "Generating certificates and keys ...")
+		}
+		if nextStepIndex == 1 {
+			out.Step(style.SubStep, "Booting up control plane ...")
+		}
+		if nextStepIndex == 2 {
+			out.Step(style.SubStep, "Configuring RBAC rules ...")
```
Comment (spowelljr) [inline pkg/minikube/bootstrapper/kubeadm/kubeadm.go:328]: Would it make more sense to just set a variable in the `if` ladder and do that `out.Step` all outside. So if we change something don't have to change it three times.

```
var msg string
if nextStepIndex == 0 {
  msg = "Generating certificates and keys ..."
}
if nextStepIndex == 1 {
  msg = Booting up control plane ..."
}
if nextStepIndex == 2 {
  msg = "Configuring RBAC rules ..."
}
out.Step(style.SubStep, msg)
```

Comment (spowelljr) [reply pkg/minikube/bootstrapper/kubeadm/kubeadm.go:328]: And maybe move this into it's own function that accepts `nextStepIndex` as a param?

Comment (medyagh) [reply pkg/minikube/bootstrapper/kubeadm/kubeadm.go:328]: aaah just tried that the `make extract` wasn't smart enough to catch it

**Comment by spowelljr** [comment]
I didn't know `make extract` was a thing. I'm just curious, you said they have to be "dumb" strings, so I'm assuming `make extract` doesn't work on strings inside structs?

**Comment by medyagh** [comment]
> I didn't know `make extract` was a thing. I'm just curious, you said they have to be "dumb" strings, so I'm assuming `make extract` doesn't work on strings inside structs?

yeah... "make extract" translates all minikube english to internationalization json words that translators can help to translate minikube https://github.com/kubernetes/minikube/tree/master/translations

@sharifelgamal added that feature to minikube

File: pkg/minikube/bootstrapper/kubeadm/kubeadm.go (hunk 1 above)
Comment (spowelljr) [reply pkg/minikube/bootstrapper/kubeadm/kubeadm.go:328]: Ah, I see

**Comment by sharifelgamal** [comment]
> I didn't know `make extract` was a thing. I'm just curious, you said they have to be "dumb" strings, so I'm assuming `make extract` doesn't work on strings inside structs?

Yeah, the extractor finds anything that called translate.T and works back. In this case, it's going to find out.Step, but anything more complex than a raw string or a string assigned to a variable is going to be hard for it to find.

**Pull request merged**